hr_expense_cancel/views/hr_expense_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="view_expenses_form_ext" model="ir.ui.view">
+            <field name="name">view.expenses.form.ext</field>
+            <field name="inherit_id" ref="hr_expense.view_expenses_form" />
+            <field name="model">hr.expense.expense</field>
+            <field name="arch" type="xml">
+                <data>                    
+                    <xpath expr="/form/header/button[@name='refuse']" position="after">
+                        <button name="cancel" states="draft,done" string="Cancel" type="workflow" groups="base.group_hr_user" />
```

Comment:
you might have an issue with security. I don't think group_hr_user have the sufficient access on account.move and account.move.line